apps/ledger-live-mobile/src/families/osmosis/UndelegationFlow/index.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React, { useMemo } from "react";
+import { Platform } from "react-native";
+import { useTranslation } from "react-i18next";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "@react-navigation/native";
+import { getStackNavigatorConfig } from "../../../navigation/navigatorConfig";
+import StepHeader from "../../../components/StepHeader";
+import { ScreenName } from "../../../const";
+import UndelegationAmount from "./01-Amount";
+import SelectDevice from "../../../screens/SelectDevice";
+import ConnectDevice from "../../../screens/ConnectDevice";
+import UndelegationValidationError from "./03-ValidationError";
+import UndelegationValidationSuccess from "./03-ValidationSuccess";
+
+const totalSteps = "3";
+
+function UndelegationFlow() {
+  const { t } = useTranslation();
+  const { colors } = useTheme();
+  const stackNavigationConfig = useMemo(
+    () => getStackNavigatorConfig(colors, true),
+    [colors],
+  );
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        ...stackNavigationConfig,
+        gestureEnabled: Platform.OS === "ios",
```

Comment:
this shouldn't go in without a comment that explain why this was needed.

Reply:
Similar to the comment above, this followed the existing Cosmos implementation: https://github.com/LedgerHQ/ledger-live/blob/c321c29c62d0ffcc7dd63d24e46444ba17e44c2d/apps/ledger-live-mobile/src/families/cosmos/UndelegationFlow/index.js#L29